akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -119,12 +128,16 @@ private[akka] object NettySSLSupport {
    * Construct a SSLHandler which can be inserted into a Netty server/client pipeline
    */
   def apply(settings: SSLSettings, log: MarkerLoggingAdapter, isClient: Boolean): SslHandler = {
-    val sslEngine = settings.getOrCreateContext(log).createSSLEngine // TODO: pass host information to enable host verification
+    val context = settings.getOrCreateContext(log)
+    val sslEngine = context.createSSLEngine()
+
+    val sslParams = new SSLParameters()
```

Comment:
Is setting the flag on `SSLParameters` superior to setting it on `sslEngine` directly?